### ISSUE TITLE
[question] about log "Translation key "myKey" has multiple different default values. Using first default"

### ISSUE THREAD

**Issue by thibaudcournollet** [i18next/i18next-scanner#69]
When I generate two translation files (en and de) after modifying the "de" file, I have this log.

**en-translations.json** : 
```
{  
  "To get started, edit <1>src/App.js</1> and save to reload.": "To get started, edit <1>src/App.js</1> and save to reload.",
  "Welcome to React": "Welcome to React"
}
```
**de-translations.json** : 
```
{
  "To get started, edit <1>src/App.js</1> and save to reload.": "Starte in dem du, <1>src/App.js</1> editierst und speicherst.",
  "Welcome to React": "Willkommen bei React und react-i18next"
}
```

**console** : 
`i18next-scanner: Translation key "To get started, edit <1>src/App.js</1> and save to reload." has multiple different default values. Using first default`

Can you explain to me why this happen with <Trans> tag and not with t() function ?

Thanks for your help.

**Comment by cheton** [comment]
Hi @wichert 

There is a warning message about default conflict that was added in this commit:
https://github.com/i18next/i18next-scanner/commit/c309f098a7d75e48b337e787c53eca87ffa9d60d#diff-6947033678b93d106e25614dd972e66f

But, there should be no default value for other languages that need to translate.

How about showing the warning message only for the first item (i.e. the default language) of the "lngs" array. That is, it will omit the warning message for 'de' and 'fr' languages with the example below:
```js
{
    lngs: ['en', 'de', 'fr'] // array of supported languages
}
```

**Comment by thibaudcournollet** [comment]
Thank you for quick response.

**Comment by cheton** [comment]
@thibaudcournollet 

This warning will not cause any issues to your current work, but we may need to suppress this warning message to prevent false positives.

**Comment by wichert** [comment]
@cheton Your suggestion definitely makes sense. I'm not sure how to accomplish that though?

**Comment by cheton** [comment]
How about adding a "defaultLng" option? If the option is missing, use the first item of the lngs array as defaultLng. Then, we can do the comparison like below:
```js
} else if ((resLoad[resKey] !== options.defaultValue) && (lng === defaultLng)) {
    // We already had a different default.
    const k = chalk.yellow(JSON.stringify(resKey));
    this.log(`i18next-scanner: Translation key ${k} has multiple different default values. Using first default`);
}
```

**Comment by cheton** [comment]
Fixed in [v2.4.1](https://github.com/i18next/i18next-scanner/releases/tag/v2.4.1)

891fa2ccbbc8d2a7026d082a021875a6ac8b9735 Add a defaultLng option for checking default values
```js
{
    defaultLng: 'en'
}
```
